MdeModulePkg/Bus/Pci/PciHostBridgeDxe/PciRootBridgeIo.c

```diff
@@ -1666,35 +1657,35 @@ RootBridgeIoAllocateBuffer (
 
   RootBridge = ROOT_BRIDGE_FROM_THIS (This);
 
-  // MU_CHANGE [BEGIN] - Allow platform to disallow unprotected DMA access when done before IOMMU install.
-  if (FeaturePcdGet (PcdRequireIommu) && (mIoMmu == NULL)) {

```

Comment:
Are we entirely reverting the PcdRequireIommu change or is it still being used anywhere? If reverting, can we do that as a separate commit (i.e. do the git revert and check this in as rebase & FF) so that we can track that for the next integration?